src/shared/components/workbasket-filters/workbasket-filters.component.ts

```diff
@@ -279,4 +280,10 @@ export class WorkbasketFiltersComponent implements OnInit {
     this.selected.caseType = undefined; // option should be blank rather than "Select a value" in case of reset.
     this.selectedJurisdictionCaseTypes = null;
   }
+
+  private getCaseFields() {
```

Comment:
Return type of `: void`? And, as @rayliangatos has said, this looks identical to the other method... a util function maybe?

Reply:
Please see my reply to @rayliangatos's comment

Reply:
Should still have a return type of `: void`, though, as per my other reply to your reply on my other comment about the similar method. 😁

Reply:
I've added a return type of void to getCaseFields in both the workbasket and search filter components. Rework on all other methods once linting is successfully incorporated into the application.